mubench.reviewsite/js/tag-util.js

```diff
@@ -0,0 +1,47 @@
+function addTag(event){
+    if (event.key === "Enter") {
```

Comment:
I'm wondering whether this event handling should be part of the handler (`onKeyUp="if (event.key == 'Enter') addTag(url, this.value)"`) and this function receive the new tag name instead of the event? I'm really not familiar with the Javascript best practices here, but this seems to mix so many different levels of responsibility (event handling,  dom handling, sending requests)...